docs/updating-the-docs.md

```diff
@@ -48,3 +46,26 @@ Examples are included in the markdown files using the [pymdown snippet notation]
 Make sure you commit both the changes to the `examples` and `databuilder/snippets` directories.
 
 If you are updating multiple examples it might be easier to the extraction step happen each time you change the python modules, you can do this with `just databuilder/watch-examples`.
+
+
+## Updating Data Builder Backend and Contract documentation
+
+!!! note "You will need both Python 3.7 and 3.10 on your PATH to make these updates"
+
+Some of the Data Builder documentation is generated from parts of databuilder's codebase.
+
+There are three components used to achieve this:
+
+* [Data Builder](https://github.com/opensafely-core/databuilder/)
```

Comment:
We refer to this as "the Data Builder repo" below. Could we do the same here? It indicates what "Data Builder" means in this context; that is, the repository.

Reply:
I've not mentioned the repo here because it's not the repo per se, but the tool itself that is the component.  I linked to the repos for each component because that's where the one can make changes.  I don't feel strongly about it though, do you think it's worth changing still?

Reply:
That's reasonable. At some point, it would be useful to note down what we're referring to with Data Builder, databuilder, etc. I don't feel strongly about it, though 🙂

Reply:
The [canonical name is `Data Builder`](https://app.shortcut.com/ebm-datalab/epic/372/rename-to-data-builder?vc_group_by=day&ct_workflow=all&cf_workflow=500000005), but I suspect many of us are referring to `databuilder` accidentally or to mean the repo without thinking about it!

Reply:
thought: I suspect there's more effort on the reader's part when trying to work out whether "Data Builder" and "databuilder" are the same or different here. My opinion rather than fact, though.

I think I'd always favour "Data Builder" unless explicitly referring to actually code use — that is, `import databuilder`. For the case of the repository, I'd probably write "Data Builder repository" and link to the repository. Anyone reading, feel free to suggest counter-examples.

(We may have inconsistencies elsewhere and we can tidy these up, before we make the documentation more public.)